src/server/room-controller.js

```diff
@@ -77,6 +89,12 @@ export default class RoomController {
       [needsPermission(permissions.CREATE_ROOMS), jsonParser, validateBody(postRoomInvitationBodySchema)],
       (req, res) => this.handlePostRoomInvitation(req, res)
     );
+
+    router.get(
+      '/api/v1/rooms/:roomId/authorize-resource-access',
+      [needsPermission(permissions.AUTORIZE_ROOMS_RESOURCES), validateParams(getAuthorizeResourceAccessParamsSchema)],
```

Comment:
If you do change this guy https://github.com/educandu/educandu/pull/110/files#r773989272 then please also change `getAuthorizeResourceAccessParamsSchema` to keep it consistent 🙏

Reply:
And `handleAuthorizeResource**s**Access` 😅

Reply:
Fixed 👍